crates/dataflow-jit/src/codegen/intrinsics/deserialize.rs

```diff
@@ -196,6 +196,26 @@ pub(super) extern "C" fn deserialize_json_date(
     }
 }
 
+pub(super) extern "C" fn deserialize_json_date_from_days(
+    place: &mut MaybeUninit<i32>,
+    json_pointer_ptr: *const u8,
+    json_pointer_len: usize,
+    map: &Value,
+) -> bool {
+    // The json pointer we're accessing the map with
+    let json_pointer = unsafe { str_from_raw_parts(json_pointer_ptr, json_pointer_len) };
+
+    if let Some(days) = map.pointer(json_pointer).and_then(Value::as_i64) {
+        // TODO: This silently truncates, do we want that?
+        place.write(days as i32);
+        false
+
+    // Otherwise the value couldn't be found and is considered null
```

Comment:
this comment (and the subsequent ones) should probably be in the else branch., with a different indentation. the empty line is also unnecessary

Reply:
The comment is on the else branch